yaks/spec/json/confucius.json_api.json

```diff
@@ -1,38 +1,18 @@
 {
-  "meta": {
-    "page": {
-      "limit": 10,
-      "offset": 0,
-      "total": 2
+  "data": {
+    "id": 9,
```

Comment:
Good catch! I already added this to https://github.com/plexus/yaks/pull/86, which will get merged once I get changes from this PR :wink: